src/main/java/com/google/devtools/build/lib/bazel/bzlmod/SingleExtensionEvalFunction.java

```diff
@@ -143,13 +143,23 @@ public SkyValue compute(SkyKey skyKey, Environment env)
     LockFileModuleExtension lockedExtension = null;
     LockfileMode lockfileMode = BazelLockFileFunction.LOCKFILE_MODE.get(env);
     if (!lockfileMode.equals(LockfileMode.OFF)) {
-      BazelLockFileValue lockfile = (BazelLockFileValue) env.getValue(BazelLockFileValue.KEY);
-      if (lockfile == null) {
+      var lockfiles =
+          env.getValuesAndExceptions(
+              ImmutableList.of(BazelLockFileValue.KEY, BazelLockFileValue.HIDDEN_KEY));
```

Comment:
I could avoid the dep on the hidden lockfile if the regular lockfile already contains the extension. Do you think that would be worthwhile? The advantage of doing it the way it's currently done is that parsing the two lockfile can be done in parallel.

Reply:
I don't think it's a big deal.